src/locales/sl/translation.json

```diff
@@ -25,38 +31,55 @@
     "subnavigation": {
       "activeDelegations": "Aktivne delegacije",
       "debondingDelegations": "Delegacije v razvezi",
+      "mobileActiveDelegations": "Delegacije",
+      "mobileDebondingDelegations": "V razvezi",
       "transactions": "Transakcije"
     },
     "summary": {
       "balance": {
         "available": "Na voljo",
         "debonding": "V razvezi",
-        "delegations": "Delegiranih",
+        "delegations": "Delegirano",
         "total": "Stanje"
       },
       "noTransactionFound": "Ni transakcij",
-      "noWalletIsOpen": "Za pošiljanje, sprejemanje, vlaganje in menjavo kovancev ROSE morate <HomeLink>odpreti vašo denarnico!</HomeLink>",
+      "noWalletIsOpen": "Za pošiljanje, sprejemanje, vlaganje in menjavo ROSE morate <HomeLink>odpreti vašo denarnico</HomeLink>!",
       "notYourAccount": "To ni vaš račun.",
       "yourAccount": "To je vaš račun."
     },
     "transaction": {
       "addEscrow": {
-        "received": "Prejeth <Amount/> v delegiranje",
-        "sent": "Delegiranih <Amount/> overitelju"
+        "received": "Prejeto <Amount> v delegiranje",
+        "sent": "Delegirano <Amount/> overitelju"
+      },
+      "consensusDeposit": {
+        "received": "Prejeto <Amount/> s pologom na ParaTime {{runtimeName}}",
+        "sent": "Polog <Amount/> na ParaTime {{runtimeName}}"
       },
+      "consensusWithdraw": {
+        "received": "Dvig <Amount/> s ParaTima {{runtimeName}}",
```

Comment:
I think these `"received"` and `"send"` translation keys are not descriptive enough.
What is their meaning?

Reply:
They are used when listing transaction history of your account. The full path of the key is actually `transaction.consensusWithdraw.received`.

Reply:
I understand that they are connected to listing an account's transaction history.

I meant to say that I don't find the `"received"` and `"sent"` very descriptive in terms of "this is the message to emit when an account is the receiving/sending account for a `consensus.Withdraw` transaction".

My ideas to make these more descriptive would be something like:
- `transaction.consensusWithdraw.sendingAccount`
- `transaction.consensusWithdraw.receivingAccount`

Or perhaps:
- `transaction.consensusWithdraw.sourceAccount`
- `transaction.consensusWithdraw.destinationAccount`

Reply:
Let's move this to another PR since it's language-agnostic.